<launch>
	<arg name="num_markers" default="4"/>
	<arg name="zero" default="false"/>
	<arg name="rate" default="100"/>
	<arg name="p" default="0.5"/>
	<arg name="noise" default="0.01"/>
	<arg name="ground_truth" default="false"/>
	<arg name="smooth" default="true"/>

	<node pkg="st_r17_calibration" name="target_publisher" type="target_publisher.py" output="screen">
		<param name="num_markers" value="$(arg num_markers)"/>
		<param name="zero" value="$(arg zero)"/>
		<param name="rate" value="$(arg rate)"/>
		<param name="p" value="$(arg p)"/>
		<param name="noise" value="$(arg noise)"/>
		<param name="ground_truth" value="$(arg ground_truth)"/>
		<param name="smooth" value="$(arg smooth)"/>
        <rosparam command="load" file="$(find st_r17_calibration)/config/dh.yaml"/>
	</node>
</launch>
